newHacks/Battle/instantWin.js

```diff
@@ -0,0 +1,2 @@
+//is a one-hit on any monster
+setInterval(() => temp1.object.n.c[0].exports.a.instance.prodigy.debugMisc.setDamageOverride(999999))
```

Comment:
Why set interval???? If you're going to use set interval. at least add a delay, so it doesn't lag the game as much.

Reply:
Agreed. Just use `modifiers`